src/components/common/CustomCard.js

```diff
@@ -51,9 +55,26 @@ function CustomCard(props) {
           ml: [3, 6],
         }}
       >
-        <Typography variant="body1" color="darkGrey.main">
+        <Typography variant="body1" color="darkGrey.main"
+          sx={{
+            display: "inline-block"
+          }}
+        >
           {title}
         </Typography>
+        
+        {tooltip && ( 
+          <Tooltip title="tooltip">
```

Comment:
Could you make the title dynamic? So we can pass in the text from the tooltip argument